source/layouts/default.haml

```diff
@@ -23,6 +23,7 @@
           - if current_article
             %a(href="javascript:window.history.back();" )= t("back")
         .col-md-4.col-sm-12
+          %br
```

Comment:
Jak chcesz mieć odstęp między kolumnami to możesz dodać `.mb-3`
https://getbootstrap.com/docs/4.1/utilities/spacing/